project/build.scala

```diff
@@ -10,7 +9,7 @@ object LunabotBuild extends Build {
   val Name = "lunabot"
   val Version = "0.1.0-SNAPSHOT"
   val ScalaVersion = "2.11.7"
-  val ScalatraVersion = "2.4.0.RC3"
+  val ScalatraVersion = "2.3.0"
```

Comment:
Add a comment in the commit that we downgraded the sclatra version because if incompatibility with scalatra-json.